runtime/primitives/src/lib.rs

```diff
@@ -59,3 +59,36 @@ pub type AuraId = sp_consensus_aura::sr25519::AuthorityId;
 
 // Moment
 pub type Moment = u64;
+
+// From https://github.com/paritytech/polkadot/pull/4332/files?diff=unified&w=1 @ runtime/common/src/lib.rs
+/// Macro to set a value (e.g. when using the `parameter_types` macro) to either a production value
+/// or to an environment variable or testing value (in case the `fast-runtime` feature is selected).
+/// Note that the environment variable is evaluated _at compile time_.
+///
+/// Usage:
+/// ```Rust
+/// parameter_types! {
+/// 	// Note that the env variable version parameter cannot be const.
+/// 	pub LaunchPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1, "KSM_LAUNCH_PERIOD");
+/// 	pub const VotingPeriod: BlockNumber = prod_or_fast!(7 * DAYS, 1 * MINUTES);
+/// }
+#[macro_export]
+macro_rules! prod_or_fast {
```

Comment:
Can you create a file like `utils.rs`, then move this macro to it?

Reply:
I was thinking we could mirror the parity PR and have a `runtime/common` module and folder, but since we have primitives already i just dropped it in there.
Instead of having another rs file in primitives (which is mostly for constants and types if i get it correctly) a new module might be cleaner, but if you prefer utils.rs i can do that too

Reply:
Maybe we should 
- rename primitives to common
- rename lib.rs to types.rs
- add utils.rs

Reply:
Did steps 2 and 3 in this revision, I like it better. If we keep adding macros in the future, we can split them out into utils.rs again